### DISCUSSION TITLE
Get the predicted values ploted in plot_model_fit

### DISCUSSION THREAD

**Discussion by TomasPapantos** [google/lightweight_mmm#280] [Q&A] [answered]
Hi, I need to get the predicted values ploted in the plot_model_fit method. 
Although I tried using the predict function, calculated the mean over it and then inverse_transform its output, the sales predicted were in many cases negative and lower than expected (not outputing the MAPE the graph shows). Does anyone know how I may get the sales used in plot_model_fit?

Thanks in advanced!

**Comment by Miguel-Adarlo** [comment, marked as answer]
What worked for me was running ``` lightweight_mmm.plot.create_media_baseline_contribution_df ```. Within, there's a column that states the average prediction. Maybe you could use that for the values.

**Reply by TomasPapantos** [reply]
Thanks Miguel, that was exactly what I was looking for. Appreciate the prompt response.
